main.py

```diff
@@ -1,7 +1,40 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    for genre_name in ("Western", "Action", "Dramma"):
+        Genre.objects.create(name=genre_name)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
```

Comment:
The actor names 'George Klooney' and 'Kianu Reaves' appear to be typos. They are later corrected, but it's better to use the correct names 'George Clooney' and 'Keanu Reeves' from the start to avoid unnecessary updates.